src/test/java/games/strategy/engine/data/AllianceTrackerTest.java

```diff
@@ -35,5 +40,22 @@ public void testAddAlliance() {
     assertTrue(relationshipTracker.isAllied(bush, castro));
   }
 
-  // TODO create test suite for Alliance/Relationships/Politics
+  @Test
+  public void getPlayersInAlliance_ShouldDifferentiateAllianceNamesThatAreSubstringsOfOtherAllianceNames() {
+    final PlayerID player1 = new PlayerID("Player1", gameData);
+    final PlayerID player2 = new PlayerID("Player2", gameData);
+    final PlayerID player3 = new PlayerID("Player3", gameData);
+    final PlayerID player4 = new PlayerID("Player4", gameData);
+    final String alliance1Name = "Alliance"; // NOTE: alliance1 name is a substring of alliance2 name
```

Comment:
instead of a note, would it be better to form `alliance2Name` as a string concatenation? It would then be enforced, eg:
```
final String alliance2Name = "Anti" + alliance1Name;
```